spread.yaml

```diff
@@ -79,6 +79,8 @@ backends:
       PermitEmptyPasswords yes
       EOF
 
+      sudo passwd -d runner
+
```

Comment:
Will try to investigate separately if we can use a real password and not rely on passwordless ssh.

Reply:
ACK to unblock CI while further investigation is following...